stories/notification-toast/props-table.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {
+    DefaultCol,
+    DescriptionCol,
+    NameCol,
+    Table,
+} from "../storybook-common/api-table";
+
+export const PropsTable = () => (
```

Comment:
Please use the new format to define the props. You may refer to files like `stories/button/props-table.tsx`. It is simpler that way. 

Also you are missing the other props in the component. We recommend to add all the props. If you are inheriting props, you should indicate them as well.